Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.props

```diff
@@ -0,0 +1,28 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IsNuGetPackage>$(MSBuildThisFileDirectory.StartsWith($(NuGetPackageRoot)))</IsNuGetPackage>
+    <!-- Referencing 'lombiq.npm.targets' using all lowercase letters, because that's the way its spelled in GitHub
+         Actions which run on Linux, which uses a case-sensitive file system. On Windows this works flawlessly. -->
+    <LombiqNpmTargetsPath
+      Condition="'$(IsNuGetPackage)' == 'true'">$(NuGetPackageRoot.TrimEnd('/\'))/lombiq.npm.targets/1.0.1-alpha.4/build</LombiqNpmTargetsPath>
```

Comment:
@Piedone Dropped using `NuGetBuild` here because we need to reference files from _Lombiq.Npm.Targets_ not only during build, but also when installed as a NuGet package.